tests/test_metrics.py

```diff
@@ -173,5 +184,82 @@ def test_largebuffer(self):
         assert m.report()['key'] == 32768 + 1
 
 
+class TestAggregators(unittest.TestCase):
+    def test_unnamed_aggregation(self):
+        report1 = {
+            'avg': AverageMetric(3, 4),
+            'sum': SumMetric(3),
+            'fixed': FixedMetric(4),
+            'global_avg': GlobalAverageMetric(3, 4),
+        }
+        report2 = {
+            'avg': AverageMetric(1, 3),
+            'sum': SumMetric(4),
+            'fixed': FixedMetric(4),
+            'global_avg': GlobalAverageMetric(1, 3),
+        }
+        agg = aggregate_unnamed_reports([report1, report2])
```

Comment:
Hmm I suppose we can safely assume that all unnamed reports will never need to be macroaveraged?

Reply:
We can implement it later as we need. For now, unnamed reports is for two cases:
- Aggregating across workers
- Aggregating across batches

Both cases will always want micro averaging.